data/lang/core/en.json

```diff
@@ -191,6 +191,10 @@
     "description": "",
     "message": "Clearance granted. Proceed to docking bay %bay."
   },
+  "CLEARANCE_GRANTED_BAY_N_GRAV_PRESS": {
+    "description": "",
+    "message": "Clearance granted. Proceed to docking bay %bay. Local gravity is %grav g, pressure is %press atm."
```

Comment:
Depending on your ambition level, one could have the last atmosphere strings as a separate string, and only append it to the rest of the message it if there actually is an atmosphere: "Atmospheric pressure is .... atm".

Just a thought.

Reply:
Done.